screens/auth/SignUpScreen.js

```diff
@@ -166,7 +168,13 @@ export default class SignUp extends React.Component {
 
     let formattedPhoneNumber = this.state.phoneNumber;
     // eslint-disable-next-line prettier/prettier
-    formattedPhoneNumber = `(${formattedPhoneNumber.slice(0, 3)}) ${formattedPhoneNumber.slice(3, 6)}-${formattedPhoneNumber.slice(6, 10)}`;
+    formattedPhoneNumber = `(${formattedPhoneNumber.slice(
+      0,
+      3
+    )}) ${formattedPhoneNumber.slice(3, 6)}-${formattedPhoneNumber.slice(
+      6,
+      10
+    )}`;
```

Comment:
This is a result of prettifying the .slice method right :0 this happens to me too (unsure why they think this is cleaner??)

Reply:
yeah HAHA this is so weird. Maybe you could outsource it to a small helper function just so it's easier to look at? 😂